public/viewjs/metrics.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Metrics Javascript
+ */
+
+/* Charting */
+var dataPoints = [];
+$("#metrics-table tbody tr").each(function () {
+	var self = $(this);
+	var label = self.find("td:eq(0)").attr('data-chart-label');
+	var value = Number(self.find("td:eq(1)").attr('data-chart-value'));
+	var dataPoint = { label: label, y: parseFloat((Math.round(value * 100) / 100).toFixed(2))};
+	dataPoints.push(dataPoint);
+});
+
+var options = {
+	exportEnabled: true,
+	legend:{
+		horizontalAlign: "center",
+		verticalAlign: "bottom"
+	},
+	data: [{
+		type: "pie",
+		showInLegend: true,
+		toolTipContent: "<b>{label}</b>: ${y} (#percent%)",
+		indexLabel: "{label}",
+		legendText: "{label} (#percent%)",
+		indexLabelPlacement: "outside",
+		valueFormatSTringt: "#,##0.##",
+		dataPoints: dataPoints
+	}]
+};
+
+// needed for recursionCount error
+recursionCount=0;
+$("#metrics-chart").CanvasJSChart(options);
+
+/* DataTables */
+var metricsTable = $('#metrics-table').DataTable({
+	"columnDefs": [
+		{ "type": "num", "targets": 1 }
+	]
+});
+$('#metrics-table tbody').removeClass("d-none");
+metricsTable.columns.adjust().draw();
+
+/* DateRangePicker */
+const urlParams = new URLSearchParams(window.location.search);
+
+var start_date = moment().startOf("month").format('MM/DD/YYYY');
+var end_date = moment().endOf("month").format('MM/DD/YYYY');
+
+if (urlParams.get('start_date')) start_date = moment(urlParams.get('start_date')) ;
+if (urlParams.get('end_date')) end_date = moment(urlParams.get('end_date'));
+
+$('#daterange-filter').daterangepicker({
+	showDropdowns: true,
+	startDate: start_date,
+	endDate: end_date,
+	locale: {
+		"format": 'MM/DD/YYYY',
```

Comment:
grocy uses fixed ISO-8601 notation for any date and time input/display ([ref](https://github.com/grocy/grocy#input-shorthands-for-date-fields)).

Reply:
corrected to `YYYY-MM-DD`

Perhaps a future issue to make this a localization preference via `config.php` setting?

Reply:
> Perhaps a future issue to make this a localization preference via `config.php` setting?

No, this is by design as [mentioned in README](https://github.com/grocy/grocy#input-shorthands-for-date-fields).